### PR TITLE
fix #1534

### LINKED ISSUE #1534: comments on patogenicity page are not saved
https://scout.scilifelab.se/cust003/18117/7df2dd00bf8311ded986d3205ef87812
for this variant I had written a comment under PS3 Functional damage (HQ), but I can not see it any longer once I go in here:

https://scout.scilifelab.se/evaluations/5dd40a3b124f63a9b5e68580

a Bug?

### PR DESCRIPTION
**Pull request by northwestwitch**
Bug fix #1534

**How to test**:
1. On current master it is possible to classify a variant using the acmg criteria and add comments and links to the classification.
1. When clicking on the classification at a later moment comments and links are not shown.
1. Install the branch and make sure that classification comment and eventual link are visible on the evaluation page but disabled. ( I don't remember how it was before but sounds logical that they can be only visualized )  

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by dNil
- [x] tests executed by dNil

### CODE REVIEW

**Comment by northwestwitch** [comment]
This ready to be reviewed!

**Comment by dnil** [comment]
The comment part worked nicely, but sth is off with the url:
Screenshot 2019-11-20 at 15 53 03
Screenshot 2019-11-20 at 15 53 10

**Comment by northwestwitch** [comment]
Thanks for the review, I'll check later today!

**Comment by northwestwitch** [comment]
It should be working now!

File: scout/server/blueprints/variant/templates/variant/acmg.html [hunk 1]
```diff
@@ -35,11 +35,19 @@ <h4>
...
               <ul class="list-group mt-3">
                 <li class="list-group-item">{{ evidence }}</li>
                 {% for criterion_code, criterion in criteria.items() %}
+
+                  {% if evaluation and evaluation.criteria.get(criterion_code).comment %}
+                    {% set comment = evaluation.criteria.get(criterion_code).comment %}
+                  {% endif %}
+                  {% if evaluation and evaluation.criteria.get(criterion_code).links %}
+                    {% set link = evaluation.criteria.get(criterion_code).links[0] %}
+                    {{ link }}
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/acmg.html]: That was debug, right?

File: scout/server/blueprints/variant/templates/variant/acmg.html [hunk 2]
```diff
@@ -51,11 +59,13 @@ <h4>Evidence of {{ category }}</h4>
                         <label for="checkbox-{{ criterion_code }}"></label>
                       </div>
                     </div>
-                    <div id="comment-{{ criterion_code }}" class="collapse {{ 'in' if evaluation and evaluation.criteria.get(criterion_code).comment }} mt-2">
+                    <div id="comment-{{ criterion_code }}" class="{{ 'collapse' if not comment }} mt-2">
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/acmg.html]: Perhaps `not (comment or link)?`

File: scout/server/blueprints/variant/templates/variant/acmg.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/acmg.html]: Yes! Thanks!!!

**Pull request merged**